sigstore/_utils.py

```diff
@@ -126,6 +131,19 @@ def base64_encode_pem_cert(cert: Certificate) -> B64Str:
     )
 
 
+def cert_der_to_pem(der: bytes) -> str:
+    """
+    Converts a DER-encoded X.509 certificate into its PEM encoding.
+
+    Returns a string containing a PEM-encoded X.509 certificate.
+    """
+
+    # NOTE: Technically we don't have to round-trip like this, since
```

Comment:
I don't understand what this comment is saying.

Reply:
I can probably just remove it: what I was trying to say was that PEM format is just `pem_headers(base64(der))`, so we _technically_ don't need to do a round-trip through the X.509 parser like this implementation is doing. However, doing the round-trip is simpler, since there's no dedicated DER-to-PEM transform API.

Reply:
Ah, gotcha. That's fine.